etc/ha_healthchecker/ha_healthchecker.py.j2

```diff
@@ -15,42 +15,29 @@ import requests
 import six
 
 
-logging.basicConfig(
-    filename="/etc/openlab/ha_healthchecker/ha_healthchecker.log",
-    filemode='a',
-    format='%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s',
-    datefmt='%H:%M:%S',
-    level=logging.DEBUG)
-LOG = logging.getLogger(__name__)
-
+# Simpledns provider
+DOMAIN_NAME = 'openlabtesting.org'
+TARGET_ORI_IP = "{{ ori_master_ip }}"
+TARGET_CHANGE_IP = "{{ slave_ip }}"
```

Comment:
This two should not be hard coded, since the cluster can be changed. Let's fix it in the next PR and add related configuration to zk as well.

Reply:
Yeah, I agree that all the original hard codes/parameters in this py file can be stored into zk.